diff --git a/components/platform/soc/bl602/bl602/bfl_main.c b/components/platform/soc/bl602/bl602/bfl_main.c
index 979a36af..6c4680a4 100755
--- a/components/platform/soc/bl602/bl602/bfl_main.c
+++ b/components/platform/soc/bl602/bl602/bfl_main.c
@@ -38,8 +38,8 @@
 #ifdef CUSTOM_LOG_IO
 HOSAL_UART_DEV_DECL(uart_stdio, 0, CUSTOM_LOG_TX_IO, CUSTOM_LOG_RX_IO, CUSTOM_LOG_RX_BAUD);
 #else
-HOSAL_UART_DEV_DECL(uart_stdio, 0, 16, 7, 2000000);
-// HOSAL_UART_DEV_DECL(uart_stdio, 0, 4, 0xff, 921600);
+// HOSAL_UART_DEV_DECL(uart_stdio, 0, 16, 7, 2000000);
+HOSAL_UART_DEV_DECL(uart_stdio, 0, 4, 0xff, 2000000);
 #endif
 
 extern uint8_t _heap_start;
